label_studio/frontend/src/pages/CreateProject/Import/Import.js

```diff
@@ -223,7 +241,13 @@ export const ImportPage = ({
     files = [...files]; // they can be array-like object
     const fd = new FormData;
 
-    for (let f of files) fd.append(f.name, f);
+    for (let f of files) {
+      if (!allSupportedFormats.includes(getFileExtension(f.name))) {
+        onError(new Error(`The filetype of file "${f.name}" is not supported.`))
```

Comment:
On a cursory glance I didn't see typographic quotes widely used in the repo. question for reviewers - would those be preferred here?